project/metals.sbt

```diff
@@ -0,0 +1,4 @@
+// DO NOT EDIT! This file is auto-generated.
+// This file enables sbt-bloop to create bloop config files.
+
+addSbtPlugin("ch.epfl.scala" % "sbt-bloop" % "1.4.0-RC1-190-ef7d8dba")
```

Comment:
Let me know if you think I should delete this file. I use VSCode for scala work so thought of adding it :)

Reply:
Metals adds this as part of the installation in the project, so let's remove it. It's usually good to add `project/metals.sbt` to your personal global `.gitignore`.